Header files "nodalview.h" and "waveformview.h" missing from Git repository? (fails to compile)

Hi again @BenPyton, interesting project! :+1: 

I tried to compile it on Linux but it fails with the following error:
```console
# cd "$(mktemp -d)"
# git clone --depth 1 https://github.com/BenPyton/SoundGenerator
# cd SoundGenerator/
# qmake5 SoundGenerator.pro
# make -j5
[..]
g++ -c -pipe -O2 -std=gnu++11 -Wall -Wextra -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_MULTIMEDIA_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -IUI -I/usr/include/qt5 -I/usr/include/qt5/QtMultimedia -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtCore -I. -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o MainWindow.o UI/MainWindow.cpp
[..]
In file included from UI/MainWindow.cpp:22:
./ui_mainwindow.h:25:10: fatal error: nodalview.h: No such file or directory
   25 | #include "nodalview.h"
      |          ^~~~~~~~~~~~~
[..]
```
It seems that files `nodalview.h` and `waveformview.h` are missing from the Git repository. Could you add them?

Thanks and best, Sebastian

--- a/UI/mainwindow.ui
+++ b/UI/mainwindow.ui
@@ -837,12 +837,12 @@ QRadioButton::indicator::checked {
   <customwidget>
    <class>NodalView</class>
    <extends>QGraphicsView</extends>
-   <header>nodalview.h</header>
+   <header>NodalView.h</header>
   </customwidget>
   <customwidget>
    <class>WaveFormView</class>
    <extends>QWidget</extends>
-   <header>waveformview.h</header>
+   <header>WaveFormView.h</header>
    <container>1</container>
   </customwidget>
   <customwidget>

